free-1.md

```diff
@@ -10,41 +10,148 @@ _What need is met by your idea? Who are you helping? What is that person's
+musician would connect oscillators and other components using wires. CSound
+takes this process and makes it digital.
+
+But there's a problem with this current state of things: DAWs operate at a high
+level, where the primitive unit is an instrument. CSound operates at a much
+lower level, where the primitive unit is an oscillator (this fact and its
+opcode-based syntax means it is used almost exclusively by academics). There is
+a lack of tools in the middle. Musicians who want to create their own
+instruments but don't want to deal with the low level of abstraction offered by
+CSound have little recourse.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+The domain of instrument creation has very clearly defined primitives at the
+level of abstraction I'm interested in. I will call these primitives sources
+and effects.
+
+A source is a sound generator, like a CSound oscillator. But a source generates
+a more complex signal than an oscillator, and is in fact a particular
+arrangement of many oscillators. A source is a pattern that can be described by
+a few parameters, but that would have to be implemented over and over again
+from the oscillator level in CSound. With only a few types of primitive sources
+-- additive and subtractive synthesizers, frequency and amplitude modulation
+synths, and physical models -- a wide variety of sounds can be created.
+
+The second type of primitive is an effect. These do not produce sound, but
+rather take a signal as input and output a different, transformed signal. In
+CSound, effects must be implemented from filters and envelopes. At a more
+intuitive (for musicians) level of abstraction, primitive effects would include
+compressors and equalizers. These effects could be chained together with
+sources to alter the sound produced by the sources.
+
+A good tool for digital music synthesis should provide these primitives, and
+should allow them to be composed by chaining in serial and in parallel. This
+already sounds like a language -- we have vocabulary (sources and effects) and
+fluency (the method for chaining).
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I have been interested in making music of all kinds for years. I took Music 88 last year, and while I enjoyed learning CSound and using it to make music, I also felt frustrated that it is so difficult to use.
+
+I was discussing potential project ideas with Daniel Zhang when the conversation turned to Melodica, and we both felt that it would be fun to make a music language like that. It seemed natural to me that there should be a language targeted at serious musicians which, provided the user is knowledgeable in the domain, is as easy to use as Melodica, but which provides more power and flexibility than a DAW. In short, I want to fill in the gap in this spectrum:
+
+Simple ---------------> Powerful
+Melodica -> DAWs -> ??? -> CSoud
```

Comment:
I like how you describe a tradeoff space and how a language can help fill that space. Often times, a DSL is a great solution for such a problem.